### ISSUE TITLE
What is an asset, and what is the difference between an asset and a device?

### ISSUE THREAD

**Issue by rafehi** [skyledge/asset-manager#3]
An asset is a thing of value. As an example, a wind turbine is an asset because it provides value to the business - it generates electricity.

A wind turbine is comprised of a multitude of parts, such as gear boxes, rotor blades and cooling units. Each of these in turn might be made of its own parts - for example, a gearbox contains gears, shafts, a control unit and a multitude of sensors to measure things like temperature and vibration.

While the top-level asset is the wind turbine, the gearbox would also be considered an asset. If we notice the gearbox vibration exceeds a threshold, it makes sense to raise a work order for maintenance/repair against the gearbox rather than the turbine, as there might be specialist repairperson to diagnose/fix/replace the gearbox.

Somewhere in this hierarchy, parts stop being assets in their own rights but rather just parts or devices. A gear within a gearbox is probably unimportant to an asset management system, other than as part of a note in a work order to get a replacement part sent in.

Are assets seperate to actual devices? If a gearbox is replaced with another gearbox, is it a new asset entirely? I would argue yes, as it is relevant to the business to know how often gearboxes are being replaced (what's the business case for more expensive gearboxes that might have a longer service life?).

**Comment by NRaf** [comment]
Yeah, I tend to agree with @viztastic on this. At the end of the day, we should empower the users to decide how best to organise data.
